src/nidigital/metadata/functions_addon.py

```diff
@@ -120,6 +120,7 @@
                 'type': 'ViInt32'
             },
             {
+                'default_value': 'datetime.timedelta(seconds=10.0)',
```

Comment:
why is this default in `_addon.py`?
why is this the only default that uses `datetime.timedelta` ?

Reply:
>why is this default in _addon.py?

Because it's part of the fancy function that's public.

> why is this the only default that uses datetime.timedelta ?

When this fancy function was developed, `timedelta` was used. Many other methods will also get the `timedelta` default when #1216 is fixed.